src/uploaders/imgur.ts

```diff
@@ -24,11 +24,10 @@ export class ImgurUploader extends UploaderTemplate {
         // !!! FILE NAME MUST BE SET, OR THE API NEVER RESPONDS !!!
         formData.append("image", options.data, options.filename);
 
-        const response = await fetch("https://api.imgur.com/3/image", {
+        const response = await fetch(`https://api.imgur.com/3/${options.type}`, {
```

Comment:
Regarding `constants.ts` - `type` doesn't really feel correct here, perhaps `endpoint`? Open for discussion IMO